src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -299,79 +303,63 @@ export class CalciteRadioButton {
   //
   // --------------------------------------------------------------------------
 
-  private renderInput(): void {
-    // Rendering a hidden radio input outside Shadow DOM so it can participate in form submissions
-    // @link https://www.hjorthhansen.dev/shadow-dom-form-participation/
-    this.input = document.createElement("input");
-    this.input.checked = this.checked;
-    this.checked && this.input.setAttribute("checked", "");
-    this.input.setAttribute("aria-label", this.value || this.guid);
-    this.input.disabled = this.disabled;
-    this.input.hidden = this.hidden;
-    this.input.id = `${this.guid}-input`;
-    if (this.name) {
-      this.input.name = this.name;
-    }
-    this.input.onblur = this.onInputBlur.bind(this);
-    this.input.onfocus = this.onInputFocus.bind(this);
-
-    // We're using option #3 explained here to hide the radio input without compromising accessibility
-    // @link https://blog.bitsrc.io/customise-radio-buttons-without-compromising-accessibility-b03061b5ba93
-    // The only difference is we're using "fixed" instead of "absolute" positioning thanks to this StackOverflow:
-    // @link https://stackoverflow.com/questions/24299567/radio-button-causes-browser-to-jump-to-the-top/24323870
-    this.input.style.opacity = "0";
-    this.input.style.position = "fixed";
-    this.input.style.zIndex = "-1";
-
-    if (this.value) {
-      this.input.value = this.value;
-    }
-    this.input.required = this.required;
-
-    if (this.el.getAttribute("title")) {
-      this.input.title = this.el.getAttribute("title");
-    } else if (this.name && this.value) {
-      this.input.title = `Radio button with name of ${this.name} and value of ${this.value}`;
-    } else {
-      this.input.title = this.guid;
-    }
-
-    this.input.type = "radio";
-    this.el.insertAdjacentElement("beforeend", this.input);
-  }
-
-  private renderLabel(): void {
-    // Rendering a calcite-label outside of Shadow DOM for accessibility and form participation
+  private renderLabel(): VNode {
     if (this.el.textContent) {
-      const textNodes = Array.from(this.el.childNodes).filter(
-        (childNode) => childNode.nodeName === "#text"
+      return (
```

Comment:
If this is the wrapping needed to avoid the maquette error, could you add a comment?

Reply:
This isn't related to maquette at all, just the new way radio renders the calcite-label.  The wrapping has to occur in the usage like `<calcite-radio-button><span>Label</span></calcite-radio-button>`

Reply:
Ah, got it. Thanks for the clarification.